tpls/gtest/gtest/gtest.h

```diff
@@ -48,6 +48,8 @@
 // registration from Barthelemy Dagenais' (barthelemy@prologique.com)
 // easyUnit framework.
 
+#define GTEST_HAS_TR1_TUPLE 0
```

Comment:
Did you consider conditionally adding the definition from CMake?

Reply:
Yes, I'm indifferent about that.

Reply:
You must 
* either clearly comment that you added that line and state why you did so (the commit message does not contain any useful information)
* or conditionally add a compile definition to the `kokkos_gtest` target

Reply:
I believe the right thing to do is the compile definition.
We already add `-DGTEST_HAS_PTHREAD=0`
https://github.com/kokkos/kokkos/blob/785d19f23ed4ba43354f9ecd7ef67cd5976ca4c8/core/unit_test/CMakeLists.txt#L17-L22